.github/workflows/rust.yml

```
@@ -79,14 +79,7 @@ jobs:
     - if: runner.os == 'Linux'
       run: sudo apt-get update && sudo apt-get -y install gcc-aarch64-linux-gnu g++-aarch64-linux-gnu libudev-dev
     - run: cargo clippy --all-targets --target ${{ matrix.sys.target }}
-    - run: make build-test
-      # TODO: enable ARM linux
-    - if: startsWith(matrix.sys.target, 'x86_64') || runner.os == 'macOS'
-      # specify directories explicitly (otherwise it will fail with missing symbols)
-      run: |
-        for I in cmd/soroban-cli cmd/crates/* cmd/crates/soroban-test/tests/fixtures/test-wasms/hello_world ; do
-          cargo test --target ${{ matrix.sys.target }} --manifest-path $I/Cargo.toml
-        done
+    - run: make test
```

The change to call `make test` instead of `cargo test --target ...` will mean when we run the tests for
```yaml
        - os: macos-latest
          target: x86_64-apple-darwin
        - os: macos-latest
          target: aarch64-apple-darwin
```
We'll actually be running the tests twice for:
```
        - os: macos-latest
          target: aarch64-apple-darwin
        - os: macos-latest
          target: aarch64-apple-darwin
```
I think we still need to specify the target.

I think we also still need to call `make build-test` to build any test wasms.

Maybe a more appropriate thing to do would be to introduce the `--workspace` in the rust.yml, like the change does in the Makefile:
```suggestion
    - run: make build-test
    - run: cargo test --workspace --target ${{ matrix.sys.target }}
```

---

Hmm I wonder why cargo is not already using an appropriate default target 🤔 is it because runners for x86-64 and ARM are cached and have both 86-64 and ARM targets?
Thanks for the catch, I fixed it with just using env variable. I think it's generally a good idea to keep logic of how we run tests in Makefile so it's easier to run it locally in the same configuration as in CI. 
> I think we also still need to call `make build-test` to build any test wasms.

Should already be called as first step of `make test` 👍🏼

---

Love it 👏🏻